lib/Marlin/Marlin/src/feature/prusa/crash_recovery.cpp

```diff
@@ -223,6 +223,7 @@ void Crash_s::update_machine() {
             #endif
             stepperX.stall_sensitivity(crash_s.sensitivity.x);
             stepperX.stall_max_period(crash_s.max_period.x);
+            tmc_enable_stallguard(stepperX);
```

Comment:
This call will undesirably clear stepperX.sfilt(filter) settings and maybe even something else set there earlier.

Reply:
Re-pushed fixed version with correct `sfilt` setting.